.test-conda-env-py3.yml

```
@@ -16,3 +16,4 @@ dependencies:
 - pyfmmlib
 - pyrsistent
 - pyvkfft
+- mpi4py
```

To be explicit about the test dependency
```suggestion
- mpi4py
- platformdirs
```

---

Good catch. Sorry this missed the merge! Is there something like `test_requires` in `setup.py`, too?

---

There was `tests_require`, but that was deprecated and removed I think. Don't think there's any other option setup.py

---

#140